ipfs-api-prelude/src/api.rs

```diff
@@ -535,6 +535,41 @@ pub trait IpfsApi: Backend {
         }
     }
 
+    /// Returns the the specified bytes of an Ipfs object.
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
I believe this doctest is failing. Add `no_run` like the other examples. These are mostly just to ensure that code can compile.

Reply:
Added.